gpu-enabler/src/main/scala/com/ibm/gpuenabler/CUDARDDUtils.scala

```diff
@@ -316,7 +403,6 @@ object CUDARDDImplicits {
                       outputArraySizes: Seq[Int] = null,
                       inputFreeVariables: Seq[Any] = null): T = {
       import org.apache.spark.gpuenabler.CUDAUtils
-
```

Comment:
Can we revert this since this change is not necessary?

Reply:
Yes. No ploblem.
Sorry to make trivial mistakes.